app/src/main/java/fr/free/nrw/commons/filepicker/PickedFiles.java

```diff
@@ -57,15 +57,12 @@ private static File tempImageDirectory(@NonNull Context context) {
      * @param file destination file
      */
     private static void writeToFile(InputStream in, File file) {
-        try {
-            OutputStream out = new FileOutputStream(file);
+        try (OutputStream out = new FileOutputStream(file)) {
             byte[] buf = new byte[1024];
             int len;
             while ((len = in.read(buf)) > 0) {
                 out.write(buf, 0, len);
             }
-            out.close();
-            in.close();
```

Comment:
While it make sense to avoid closing inputstream provided by the caller, it doesn't seems appropriate to leave it unclosed right ?

Could you kindly tweak the callers so that they're closed appropriately?